Vagrantfile

```diff
@@ -2,20 +2,42 @@
 # vi: set ft=ruby :
 
 Vagrant.configure(2) do |config|
-  config.vm.box = "jonasrosland/rexray"
 
-  # Add a SATA controller with 30 ports to the VM, so REX-Ray can add disks on the fly
-  config.vm.provider :virtualbox do |vb|
-   vb.customize ["storagectl", :id, "--add", "sata", "--controller", "IntelAhci", "--name", "SATA", "--portcount", 30, "--hostiocache", "on"]
-  end
+  # Amount of nodes to start
+  nodes = 2
+
+  (1..nodes).each do |i|
+    config.vm.define "rexray-#{i}" do |node|
+      node.vm.box = "jonasrosland/rexray"
+
+      # Add a SATA controlle with 30 ports to the VM, so REX-Ray can add disks on the fly
+      node.vm.provider :virtualbox do |vb|
+       vb.customize ["storagectl", :id, "--add", "sata", "--controller", "IntelAhci", "--name", "SATA", "--portcount", 30, "--hostiocache", "on"]
+       vb.customize ["modifyvm", :id, "--macaddress1", "auto"]
+      end
+
+      # Set the current $PWD as the place where you will
+      # store the VMDKs that are attached to the VM.
+      dir = "#{ENV['PWD']}/Volumes"
+
+      node.vm.provision "shell", inline: <<-SHELL
+        ## Optionally remove REX-Ray
+        #rpm -e rexray
 
-  # Set the current $PWD as the place where you will
-  # store the VMDKs that are attached to the VM.
-  # Ugly code and will work on a fix later.
-  dir = "echo #{ENV['PWD']} | sed 's/\\\//\\\\\\//g'"
-  config.vm.provision "shell", inline: <<-SHELL
-    #{dir} > /root/dir.txt
-    sed -i 's/.*volumePath.*/  volumePath: '$(cat /root/dir.txt)' /g' /etc/rexray/config.yml
-    rm -f /root/dir.txt
-  SHELL
+        ## Optionally get latest stable REX-Ray
+        #sudo curl -sSL https://dl.bintray.com/emccode/rexray/install | sh -s stable
+
+        ## Optionally get latest release candidate REX-Ray
+        #sudo curl -sSL https://dl.bintray.com/emccode/rexray/install | sh -s staged
+
+        ## Optionally update the volume path for pwd Volumes dir
+        sed -i '/.*volumePath.*/c\\\x20\x20volumePath: \"#{dir}\"' /etc/rexray/config.yml
+
+        ## Optionally set preemption
+        #sed -i '/.*preempt.*/c\\\x20\x20\x20\x20\x20\x20preempt: true' /etc/rexray/config.yml
+
+        service docker start 2&>1
```

Comment:
This commands gives me the following Vagrant output:
```
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
```
If you change it to
`service docker restart`
it will give you this output and not error out:
```
==> rexray-1: Redirecting to /bin/systemctl restart  docker.service
```